pytest_zigzag/data/configs/jenkins-config.json

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
Im not sure this list is complete.  I think that all the ENVs that we capture for mk8s are all jenkins ENVs.

Reply:
This is where I got the env vars from: https://wiki.jenkins.io/display/JENKINS/Building+a+software+project#Buildingasoftwareproject-belowJenkinsSetEnvironmentVariables